libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.cpp

```diff
@@ -718,6 +718,48 @@ void wifi_dns_found_callback(const char *name, const ip_addr_t *ipaddr, void *ca
     esp_schedule(); // break delay in hostByName
 }
 
+void wifi_dns_found_callback_async(const char* name, const ip_addr_t* ipaddr, void* callback_arg);
+
+/**
+ * Resolves the given hostname to an IP address asynchronously.
+ * @param aHostname     Name to be resolved
+ * @param aResult       IPAddress structure to store the returned IP address
+ * @param waiting       waiting flag if DNS search is still in progress
+ * @return 1 if aHostname was successfully converted to an IP address,
```

Comment:
`if a*space* hostname...` and the alignment seems kind of funky, too.

Reply:
It's the same description as ESP8266WiFiGenericClass::hostByName.
I just copy-pasted and added a line for waiting parameter.